html/elements/script.json

```diff
@@ -603,6 +603,43 @@
                 "standard_track": true,
                 "deprecated": false
               }
+            },
+            "integrity": {
+              "__compat": {
+                "description": "<code>integrity</code>",
+                "spec_url": "https://github.com/whatwg/html/pull/10269",
```

Comment:
Do we need to wait for the PR to land?

Reply:
Ideally, yes. If it is not in a spec yet, it can still go on MDN, but it needs to be marked as non-standard (`"standard_track": false`) and have the `"spec_url"` field removed. Then later on when it does land, we'll need to a little more updating.

It depends on how quickly you want to get it on MDN, and when the spec PR will be merged.

Reply:
The spec PR is expected to land in a week. It's ready to go, but Mozilla folks added to hold off on landing the PR until they reviewed. Seems better to wait a week here than to land it as non-standard.

Reply:
PR landed